cmd/cloudflared/service_template.go

```diff
@@ -48,10 +49,17 @@ func (st *ServiceTemplate) Generate(args *ServiceTemplateArgs) error {
 	if err != nil {
 		return fmt.Errorf("error generating %s: %v", st.Path, err)
 	}
-	fileMode := os.FileMode(0644)
+	fileMode := os.FileMode(0o644)
 	if st.FileMode != 0 {
 		fileMode = st.FileMode
 	}
+
+	plistFolder := path.Dir(resolvedPath)
+	err = os.MkdirAll(plistFolder, 0o755)
+	if err != nil {
+		return fmt.Errorf("error creating %s: %v", plistFolder, err)
```

Comment:
Could this also error if the directories exist? Should we ignore `IsExists` error?

Reply:
No, MkdirAll returns `nil` if the directory exists, similar to `mkdir -p`. 

https://pkg.go.dev/os#MkdirAll